### DISCUSSION TITLE
Linux port possible?

### DISCUSSION THREAD

**Discussion by majal** [p0deje/Maccy#867] [Ideas]
Maccy, inspired by [Parcellite](http://parcellite.sourceforge.net/), is much better than Parcellite at this point. 🎉

Can we port back to Linux? 😊

**Comment by p0deje** [comment]
When 2.0 is complete, it should be technically possible to implement a Linux port since SwiftUI is supposed to compile into GTK (https://www.swift.org/blog/adwaita-swift/). Many implementation details would have to be reworked of course (e.g. system clipboard interaction). If anyone is up to try working on it, I'd support that.

**Reply by majal** [reply]
Sounds nice. Thank you for the beautiful work you did here. I don't have the resources to help on this one, but one thing I can do now is to suggest it to friends and promote it.

Again, thank you for your good work!
